lib/semantics/semantics.cc

```diff
@@ -208,6 +209,70 @@ void SemanticsContext::PopConstruct() {
   constructStack_.pop_back();
 }
 
+void SemanticsContext::SayDoVarRedefine(
+    const Symbol &variable, const parser::CharBlock &location) {
```

Comment:
Other "Say" functions that take a `CharBlock` for the location of the message have that as the first argument, so it would be consistent.